build-logic/packaging/src/main/kotlin/gradlebuild/run/tasks/RunEmbeddedGradle.kt

```diff
@@ -24,6 +24,7 @@ import org.gradle.api.tasks.Classpath
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.options.Option
+import org.gradle.kotlin.dsl.*
```

Comment:
:x: `*` import.

Reply:
Added a comment that this workarounds https://youtrack.jetbrains.com/issue/KTIJ-24390/